wallet/test/widget_testing/extension/size_extension.dart

```diff
@@ -28,11 +31,32 @@ extension SetScreenSize on WidgetTester {
         child: ScreenUtilInit(
           minTextAdapt: true,
           builder: (_, __) => MaterialApp(
-            // key: UniqueKey(),
             navigatorKey: navigatorKey,
             debugShowCheckedModeBanner: false,
             title: "Pylons Wallet",
             theme: PylonsAppTheme().buildAppTheme(),
+            routes: {
+              RouteUtil.ROUTE_PURCHASE_VIEW: (context) {
```

Comment:
Whats the need for adding it here?

Reply:
As i write test case for tap button in accept policy screen, on tap of that button it is actually calling `Navigator.pushNamed` and in test case it didn't find any route in MaterialApp and test was failing, so for that we have to defined it in here too.